src/pybind/mgr/status/module.py

```diff
@@ -183,7 +183,7 @@ def handle_fs_status(self, cmd):
         output += "\n" + standby_table.get_string() + "\n"
 
         if len(mds_versions) == 1:
-            output += "MDS version: {0}".format(list(mds_versions))[0])
```

Comment:
In 13.2.2 it is using *mds_versions.keys()* https://github.com/starlingx-staging/stx-ceph/blob/ddfb3f76cc29f499fa13730e8ff6335d80cdc3b2/src/pybind/mgr/status/module.py#L243

I see in 14.2.22 https://github.com/starlingx-staging/stx-ceph/blob/83c70bf033cd6f7b9ea1a1f96023dcfcb912f977/src/pybind/mgr/status/module.py#L186 this has changed but syntax is wrong.

How did you verify this syntax was wrong ?

Reply:
You can check this with 'ceph -s' command.